src/lib/components/particles/circle-icon/index.jsx

```diff
@@ -0,0 +1,15 @@
+export const CircleIcon = ({color}) => {
+
+    return (
+        <svg
+            style={{ display: 'inline-block', marginRight: 4, transform: 'translateY(-1px)' }}
```

Comment:
I think these styles are actually going to be pretty standard, given the context of this as going into pieces of text. Would be a bit of a faff to add a margin to this each time so that it doesn't overlap the lettering which we will never want. So I would suggest keeping these here then adding a local override where needed?

Reply:
but i can put them in a css file